WordPress/src/main/java/org/wordpress/android/ui/posts/PrepublishingHomeViewModel.kt

```diff
@@ -91,7 +96,8 @@ class PrepublishingHomeViewModel @Inject constructor(
                                 ?: run { UiStringRes(R.string.prepublishing_nudges_home_tags_not_set) },
                         actionClickable = true,
                         onActionClicked = ::onActionClicked
-                ))
+                )
```

Comment:
small nitpick this should be moved to the right

Reply:
I mean - indented

Reply:
Thanks! @mzorz is there any way that the checkstyle tooling can be improved to catch things like these? It's sometimes hard to spot them easily.

Reply:
I'm sure there is, but it's a nitpick anyway. Given we have other priorities right now, we can leave as is 👍